### DISCUSSION TITLE
FlareSolverr doesn't work when used in combination with another HTTP client

### DISCUSSION THREAD

**Discussion by RudraPSinha** [FlareSolverr/FlareSolverr#779] [Q&A]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.1.2
- Last working FlareSolverr version: NA
- Operating system: Windows 64x
- Are you using Docker: no
- FlareSolverr User-Agent (see log traces or / endpoint): NA
- Are you using a VPN: no
- Are you using a Proxy: no
- Are you using Captcha Solver: no
- If using captcha solver, which one: NA
- URL to test this issue: https://betproexch.com
```

### Description

I have wrote a little script in nodejs which uses axios to make a request to the URL using the response that was received from the FlareSolverr. The initial challenge is passed but when the cookie and the user agent is passed to the axios http client for subsequent requests it again throws a challenge, which it probably shouldn't as a warning was suggested in the readme which says that if user-agent is added to the header it should not throw the challenge: 

````
const axios = require('axios');
const post_body = {
  "cmd": "request.get",
  "url":"https://URL",
  "session":"ALPHA1",
  "maxTimeout": 60000
}
axios.post('http://localhost:8191/v1', post_body, {
    headers: {
        'Content-Type': 'application/json'
    }
}).then((response) => {
    if (response.status === 200) {
        const json_response = response.data;
       
        if (json_response.status === 'ok') {
            const cookies = json_response.solution.cookies;
            const clean_cookies_dict = cookies.reduce((acc, cookie) => ({...acc, [cookie.name]: cookie.value}), {});
            const user_agent = json_response.solution.userAgent;
            console.log(user_agent, clean_cookies_dict)
            axios.get('https://URL', {
                headers: {
                    'User-Agent': user_agent,
                    cookies: clean_cookies_dict
                },
                withCredentials: true,
                cookies: clean_cookies_dict
            }).then((response) => {
                if (response.status === 200) {
                    // ...parse data from response
                    console.log(response.data,"Cem")
                    //console.log('Success');
                }
            }).catch((error) => {
                console.log(error.response.data)
                console.error(error.message);
            });
        }
    }
}).catch((error) => {
    console.error(error);
});
````

### Logged Error Messages

```text
NA
```

**Comment by pinyugi** [comment]
You do realize there is such thing as a TLS/HTTP2.0 fingerprint, changing your user-agent in 2023 is irrelevant.
Cloudflare is obviously detecting the request comes from a different client than the one that originally requested it(the browser), regardless if you pass in the cookies from FlareSolver and change your user-agent on your axios client.

Your connection fingerprints obviously do not match, change them, look for open source projects that can change your connection fingerprints.

**Comment by ngosang** [comment]
That code should work but I think you are building the cookie header wrong. The format must be something like this:
```
Cookie: name=value; name2=value2; name3=value3
```
